### PR TITLE
Clarify that ClientRequest.from(..) also copies body

### PR DESCRIPTION
**Pull request by AlexejTimonin**
- Add to javadoc that ```ClientRequest.from(..)``` will also copy body from the given request.
- Add test to show that the body inserter will be copied over.

### CODE REVIEW

File: spring-webflux/src/main/java/org/springframework/web/reactive/function/client/ClientRequest.java [hunk 1]
```diff
@@ -125,8 +125,8 @@
 	// Static builder methods
 
 	/**
-	 * Create a builder with the method, URI, headers, and cookies of the given request.
-	 * @param other the request to copy the method, URI, headers, and cookies from
+	 * Create a builder with the method, URI, headers, cookies and body of the given request.
+	 * @param other the request to copy the method, URI, headers, cookies and body from
```
Comment (qiangmao) [inline spring-webflux/src/main/java/org/springframework/web/reactive/function/client/ClientRequest.java:129]: 这是一个评论

Comment (sbrannen) [inline spring-webflux/src/main/java/org/springframework/web/reactive/function/client/ClientRequest.java:129]:
```suggestion
	 * Create a builder with the method, URI, headers, cookies, attributes, and body of the given request.
	 * @param other the request to copy the method, URI, headers, cookies, attributes, and body from
```

**Review by sbrannen — CHANGES_REQUESTED** [review]
I've requested one small change to document that "attributes" are copied as well.

Please also update the copyright date in `ClientRequest` and `DefaultClientRequestBuilderTests`.

File: spring-webflux/src/main/java/org/springframework/web/reactive/function/client/ClientRequest.java (hunk 1 above)
Comment (AlexejTimonin) [reply spring-webflux/src/main/java/org/springframework/web/reactive/function/client/ClientRequest.java:129]: Thank you for your review and suggested changes. I will address them this evening.

**Comment by AlexejTimonin** [comment]
@sbrannen I could not see any tests for attributes in ```DefaultClientRequestBuilderTests``` so I updated the ```from()``` test to include attributes.

File: spring-webflux/src/main/java/org/springframework/web/reactive/function/client/ClientRequest.java (hunk 1 above)
Comment (AlexejTimonin) [reply spring-webflux/src/main/java/org/springframework/web/reactive/function/client/ClientRequest.java]: I don´t understand your review, could you take it in English?

**Review by sbrannen — APPROVED** [review]

**Pull request closed without merging**